### ISSUE TITLE
Add GitHub pages link?

### ISSUE THREAD

**Issue by mckelveygreg** [hatbearingoctopus/Book-List#1]
Have you used GitHub pages before? It is an easy way to actually host your project! If love to see what it looks like! I used them to host all of my side projects

**Comment by hatbearingoctopus** [comment]
I haven't, I'm still getting used to just getting my code onto GitHub. I will look into this and try to get them hosted this way. Thanks!

**Comment by mckelveygreg** [comment]
Maybe our next meetup, I could help you out! Let me know!

**Comment by hatbearingoctopus** [comment]
@mckelveygreg Thanks for giving me a hand yesterday! I'm slowly getting everything hosted through GitHub and finally got my portfolio linked to my profile etc. Baby steps!

**Comment by mckelveygreg** [comment]
yeah!! It feels so great to actually get things on the world wide web!!
